package.json

```diff
@@ -29,14 +29,15 @@
     "url": "https://github.com/ipfs/is-ipfs.git"
   },
   "scripts": {
+    "prepare": "aegir build --no-bundle",
     "test:node": "aegir test --target node",
     "test:browser": "aegir test --target browser",
     "test": "aegir test",
     "lint": "aegir lint && aegir lint-package-json",
     "release": "aegir release",
     "release-minor": "aegir release --type minor",
     "release-major": "aegir release --type major",
-    "build": "aegir build",
+    "prepublishOnly": "aegir build",
     "coverage": "aegir coverage",
     "coverage-publish": "aegir coverage --upload"
```

Comment:
we should remove these, aegir doesnt have them for a long time

Reply:
Done.